test/test_torch.py

```diff
@@ -18247,7 +18301,7 @@ def test_count_nonzero(self, device, dtype):
         self._test_reduction_function_with_numpy(torch.count_nonzero, np.count_nonzero, device, dtype)
         self._test_reduction_function_with_numpy(torch.count_nonzero, np.count_nonzero, device, dtype, True)
 
-    def _test_sum_reduction_vs_numpy(self, torch_fn, np_fn, device, dtype, with_keepdim=False, with_extremal=False):
+    def _test_nanfunc_reduction_vs_numpy(self, torch_fn, np_fn, device, dtype, with_keepdim=False, with_extremal=False):
```

Comment:
Earlier, this function was used only for `nansum`. This function name can be discussed, should it be `_test_sum_prod_reduction_vs_numpy` instead? @mruberry

Reply:
This name seems fine.